transactions/set-version-boundary/2023/sep-18/arguments.json

```
@@ -0,0 +1,22 @@
+[
+  {
+    "type": "UInt8",
+    "value": "0"
+  },
+  {
+    "type": "UInt8",
+    "value": "31"
+  },
+  {
+    "type": "UInt8",
+    "value": "17"
+  },
+  {
+    "type": "Optional",
+    "value": "patch.1"
```

Not entirely sure as I don't work with Cadence JSON much, but I think this should be:
```suggestion
    "value": {
      "type": "String,
      "value": "patch.1"
```
Based on [these docs](https://developers.flow.com/cadence/json-cadence-spec#optional) for non-nil optional values

---

updated...thanks for flagging it.